selenium_tests/course_admin/page_objects/course_admin_dashboard_page_object.py

```diff
@@ -26,9 +27,30 @@ def manage_people_button_is_displayed(self):
             return False
         return True
 
+    def class_roster_button_is_displayed(self):
+        """
+        Verifies that the Class Roster button is displayed
+        """
+        try:
+            self.focus_on_tool_frame()
+            self.find_element(*Locators.CLASS_ROSTER_BUTTON)
```

Comment:
This only checks the DOM for the button; it might return the element even if the element is hidden. I think we want something like http://selenium-python.readthedocs.io/api.html#selenium.webdriver.remote.webelement.WebElement.is_displayed after getting the element reference as well.